com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -205,6 +205,22 @@ public class NetworkSceneManager : IDisposable
 
         internal Scene DontDestroyOnLoadScene;
 
+        /// <summary>
+        /// LoadSceneMode.Single: All currently loaded scenes on the client will be unloaded and
+        /// the server's currently active scene will be loaded in single mode on the client
+        /// unless it was already loaded.
+        ///
+        /// LoadSceneMode.Additive: All currently loaded scenes are left as they are and any newly loaded
+        /// scenes will be loaded additively.  Users need to determine which scenes are valid to load via the
+        /// <see cref="VerifySceneBeforeLoading"/> method.
+        /// </summary>
+        public LoadSceneMode ClientSynchronizationMode { get; internal set; }
```

Comment:
tests for Single & Additive to validate this works in both modes and gives us the results we expect on the client-side? 👀

Reply:
Well, for single mode it works already otherwise it wouldn't have passed the already existing tests.
I will add some additional passes in the existing tests which use additive ClientSynchronizationMode.

Reply:
Added to the scene loading tests.  It now runs the tests with client synchronization set to both LoadSceneMode.Additive and LoadSceneMode.Single.